sktime/util.py

```diff
@@ -1,15 +1,38 @@
 import collections
 import itertools
 import numbers
+import os
 from typing import Optional, List
 from weakref import WeakKeyDictionary
 
 import numpy as np
 
 
-def handle_n_jobs(value: Optional[int]):
+def handle_n_jobs(value: Optional[int]) -> int:
+    r"""Handles the n_jobs parameter consistently so that a non-negative number is returned.
+    In particular, if
+
+      * value is None, use 1 job
+      * value is negative, use number cores available * 2
```

Comment:
0_o

Reply:
This yields better load-balancing and makes better use of hyperthreading

Reply:
cool